packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -202,6 +202,9 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         // Must always receive the data store type inside of the attributes
         if (tree?.trees !== undefined) {
             Object.keys(tree.trees).forEach((path) => {
+                // Issue #4414
+                if (path === "_search") { return; }
```

Comment:
Where does this search blob come from? Does server write it?

Reply:
Bohemia writes it. That's the reason this code ideally should not be here. I.e. this is temp solution, I'll keep an issue opened and will remove it after Arin finishes his work to separate snapshot paths for user data (like _search) vs. channels (DDS)

Reply:
Yeah, that makes sense. Can you please add a link to the issue here.